pkg/deployment/service.go

```diff
@@ -61,8 +61,8 @@ func DeployService(ctx context.Context, helper *helper.Helper, obj client.Object
 
 // GetServices returns the list of services for the node's role
 // Note that these are not inherited from NodeTemplate.
-func GetServices(instance *dataplanev1.OpenStackDataPlaneNode,
-	instanceRole *dataplanev1.OpenStackDataPlaneRole) []string {
+func GetServices(instance *dataplanev1.OpenStackDataPlaneNodeSet,
+	instanceRole *dataplanev1.OpenStackDataPlaneNodeSet) []string {
```

Comment:
this doesn't look right. This function may not be used any longer since node_controller is dropped.

Reply:
I'll check, it looks like this was probably something that was just changed with my `%s/OpenStackDataPlaneRole/OpenStackDataPlaneNodeSet/g`. I'll take a closer look at it.